servers.json

```diff
@@ -1,236 +0,0 @@
-{
-  "apps": [
```

Comment:
do we need to update the docs that `servers.json` is not a thing anymore? or should we take `mysql_servers` and call it `servers` now?